docs/tutorial/part-five/index.md

```diff
@@ -91,7 +91,7 @@ Space</kbd>).
 
 ![filesystem-autocomplete](filesystem-autocomplete.png)
 
-Try adding a number of fields to your query, pressing <kbd>Ctrl + Enter</kbd>
+Try adding a number of fields to your query pressing, <kbd>Ctrl + Enter</kbd>
```

Comment:
This seemed correct earlier as well! Sorry for the back and forth!

Reply:
Yeah, no problem.